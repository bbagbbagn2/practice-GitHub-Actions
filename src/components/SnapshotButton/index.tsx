// src/components/SnapshotButton/index.tsx

export default function SnapshotButton({
  children,
}: {
  children: React.ReactNode;
}) {
  return <button className="bg-red-500">{children}</button>;
}
