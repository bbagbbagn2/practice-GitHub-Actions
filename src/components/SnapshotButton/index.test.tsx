// src/components/SnapshotButton/index.test.tsx

import { render } from "@testing-library/react";
import SnapshotButton from ".";

test("Button 컴포넌트가 올바르게 렌더링됩니다", () => {
  const { container } = render(<SnapshotButton>클릭하세요</SnapshotButton>);
  expect(container).toMatchSnapshot();
});
