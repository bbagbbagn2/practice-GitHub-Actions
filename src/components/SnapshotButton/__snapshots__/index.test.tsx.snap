// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`Button 컴포넌트가 올바르게 렌더링됩니다 1`] = `
<div>
  <button
    class="bg-red-500"
  >
    클릭하세요
  </button>
</div>
`;
